testutils/check-baselines.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -e
+
+# TODO: We should figure out how to do this as part of GapicGeneratorTest at some point.
+
+exitStatus=0
+
+## Check for unhandled cases.
+unhandledStr='$unhandledCallingForm'
+if grep --include='*.baseline' -r -l -F $unhandledStr src/; then
+	cat 1>&2 <<EOF
+Above baseline files contain string "$unhandledStr";
+some sample code won't render properly.
+
+EOF
+	exitStatus=1
+fi
+
+## Check that every calling form is used somewhere.
+callingFormsFile="src/main/java/com/google/api/codegen/viewmodel/CallingForm.java"
+
+# Calling forms are in an enum, so we look for capital identifiers that begins the line.
+callingForms=$(< $callingFormsFile grep -o '^[[:space:]]*[A-Z][a-zA-Z]*' | tr -d -c '[a-zA-Z\n]')
+
+unusedForms=$(
+	# Grep src/
+	#   recursively (-r)
+	#   in baseline files (--include='*.baseline')
+	#   for any calling forms: -f <(cat <<< $callingForms)
+	#   use fixed strings, not regexp (-F)
+	#   match whole word (-w)
+	#   print only the calling form, not surrounding line (-o)
+	#   don't print file names (-h)
+	# Prints all occurences of used calling forms
+	grep --include='*.baseline' -h -w -o -r -F -f <(cat <<< $callingForms) src/ |
+
+	# Then turn it into a set.
+	sort -u |
+
+	# Both callingForms and used forms are sets, sorting them together means
```

Comment:
Rephrase a bit:
"Both $callingForms  and the used forms determined so far in this pipeline are sets, so sorting ...."

Reply:
Done